Buildkite/ephemeral-agent.md

```diff
@@ -0,0 +1,119 @@
+
+1. Install and configure a Buildkite agent on a machine that has network visibility to your Orka environment. For more information about how to install and configure a Buildkite agent, see the [instructions][agent-instructions] provided by Buildkite.
+2. Copy the provided scripts to the agent machine: [base.sh](scripts/base.sh), [bootstrap.sh](scripts/bootstrap.sh), [run.sh](scripts/run.sh). 
+**Note**: All scripts should be in the same directory. For example, you can add them to `/usr/local/var/buildkite-agent/`. Overwrite the existing `bootstrap.sh` file if asked.
+3. Make the scripts executable by running `chmod +x path_to_script` in the command line.
+4. Copy the provided [pre-exit hook](scripts/pre-exit) to the agent machine. You should add it to the agent hooks directory. It can be found in the `buildkite-agent.cfg` file under the `hooks-path` property. The location of the `buildkite-agent.cfg` varies depending on your OS. For more information, see your [platform's installation instructions][agent-instructions].
+5. Make the hook executable by running `chmod +x path_to_hook` in the command line.
+6. Verify [jq][jq] is installed. For more information, see [jq][jq] page.
+7. Verify that the machine has network visibility to the Orka environment. If the machine is part of the Orka environment, skip this step. You can use any VPN client to connect to the Orka environment. For more information, see your Orka [IP Plan][ip-plan].
+8. Verify that the private SSH key created [earlier](#set-up-an-orka-vm-base-image) that is used to connect to the ephemeral agent is present on the wrapper machine and added to the ssh-agent.
+9. Verify that the private SSH keys used access the code repositories used by the build job are present on the wrapper machine and added to the ssh-agent.  
+**Note** For more information about using multiple SSH keys, see [here][multiple-ssh-keys].
+
+## Buildkite environment variables
+
+The provided scripts expect the following environment variables to be set:
+
+* `ORKA_USER` - User used to connect to the Orka environment. Created by running `orka user create`
+* `ORKA_PASSWORD` - Password used to connect to the Orka environment. Created by running `orka user create`
+* `ORKA_ENDPOINT` - The Orka endpoint. By default it is `http:/10.10.10.100`
+* `ORKA_VM_NAME` - The name of the VM to be deployed. This should match the VM config created [earlier](#set-up-an-orka-vm-config-for-the-ephemeral-agents)
+* `ORKA_VM_USER` - User used to SSH to the VM
+
+## Using the Buildkite agent
+
+Once the setup of the Buildkite agent is finished, you can run your CI/CD pipelines in Orka.
+
+## Connectivity
+
+The communication between the Buildkite agent and the Buildkite server is instantiated from the agent.
+
+This means your Orka environment must have visibility to the Buildkite server.
+
+Visibility from the Buildkite server to the Orka environment is not required. 
+
+## Advanced: Changing the Buildkite environment variables
```

Comment:
Maybe `Advanced: Additional Buildkite environment variables`? The current title no longer makes sense with the section title from a few sections back.